src/command_validate.cc

```diff
@@ -22,48 +27,97 @@ auto sourcemeta::jsonschema::cli::validate(
   const auto options{
       parse_options(arguments, {"h", "http", "b", "benchmark", "t", "trace"})};
 
-  if (options.at("").size() < 1) {
+  auto leftover = options.at("");
+
+  // Attempt to extract a pointer value from leftover arguments:
+  //    --path=<pointer>
+  //    --path <pointer>
+  //    -p <pointer>
+  std::optional<std::string> pointer_option;
+  for (auto it = leftover.begin(); it != leftover.end();) {
+    const auto &arg = *it;
+
+    // a) --path=/something
+    if (arg.rfind("--path=", 0) == 0) {
```

Comment:
I believe `options.at("p")` and `options.at("path")` will give you what you want? Look at how we take options in other commands, like for `--resolve`